src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -317,7 +317,9 @@ class AssemblerConnectionManager(
     }
 
     fun dropOrganizationDatabase(organizationId: UUID) {
-        dropOrganizationDatabase(organizationId, organizations.getOrganizationDatabaseName(organizationId))
+        dropOrganizationDatabase(
+                organizationId,
+                extDbManager.getOrganizationDatabaseName(organizationId))
     }
 
     fun dropOrganizationDatabase(organizationId: UUID, dbName: String) {
```

Comment:
is this ever called from somewhere else than the above `dropOrganizationDatabase` function? That feels a little sketchy to me since we should be strictly enforcing `dbName` which is used in the DROP DATABASE call being computed based on `organizationId`

Reply:
Not anymore, this is one of those ones I tried to change earlier but had to rework after the merges